webpack.common.js

```diff
@@ -19,46 +20,51 @@ module.exports = {
-                            'dynamic-import-webpack', // Needed for dynamic imports.
-                            'lodash', // Automagically tree-shakes lodash.
-                            'transform-regenerator', // Transforms async and generator functions.
+                            '@babel/plugin-syntax-dynamic-import', // add support for dynamic imports (used in app.js)
+                            'lodash', // Tree-shake lodash
                         ],
                         presets: [
-                            ['env', {
-                                loose: true, // Enable "loose" transformations for any plugins in this preset that allow them.
-                                modules: false, // Don't transform modules; needed for tree-shaking.
-                                useBuiltIns: true, // Tree-shake babel-polyfill.
+                            ['@babel/preset-env', {
+                                loose: true, // Enable "loose" transformations for any plugins in this preset that allow them
+                                modules: false, // Don't transform modules; needed for tree-shaking
+                                useBuiltIns: 'usage', // Tree-shake babel-polyfill
+                                targets: '> 1%, last 2 versions, Firefox ESR',
                             }],
                         ],
                     },
                 },
             },
-            {
-                test: /jquery-migrate/,
-                use: 'imports-loader?define=>false',
-            },
         ],
     },
     output: {
         chunkFilename: 'theme-bundle.chunk.[name].js',
         filename: 'theme-bundle.[name].js',
         path: path.resolve(__dirname, 'assets/dist'),
     },
+    performance: {
+        hints: 'warning',
+        maxAssetSize: 1024 * 150,
+        maxEntrypointSize: 1024 * 150,
+    },
     plugins: [
         new CleanPlugin(['assets/dist'], {
             verbose: false,
             watch: false,
         }),
         new LodashPlugin, // Complements babel-plugin-lodash by shrinking its cherry-picked builds further.
-        new webpack.ProvidePlugin({
+        new webpack.ProvidePlugin({ // Provide jquery automatically without explicit import
             $: 'jquery',
             jQuery: 'jquery',
             'window.jQuery': 'jquery',
         }),
+        new BundleAnalyzerPlugin({
+            analyzerMode: 'static',
+            openAnalyzer: false,
+        }),
     ],
     resolve: {
         alias: {
-            'jquery-migrate': path.resolve(__dirname, 'node_modules/jquery-migrate/dist/jquery-migrate.min.js'),
+            jquery: path.resolve(__dirname, 'node_modules/jquery/dist/jquery.min.js'),
             jstree: path.resolve(__dirname, 'node_modules/jstree/dist/jstree.min.js'),
             lazysizes: path.resolve(__dirname, 'node_modules/lazysizes/lazysizes.min.js'),
             pace: path.resolve(__dirname, 'node_modules/pace/pace.min.js'),
```

Comment:
Should swap the Pace alias for Nanobar.

`nanobar: path.resolve(__dirname, 'node_modules/nanobar/nanobar.min.js'),`

Reply:
Good catch, thanks!